packages/shared/src/utils/cfd/available-cfd-accounts.ts

```diff
@@ -0,0 +1,37 @@
+import { localize } from '@deriv/translations';
+import { CFD_PLATFORMS } from '../platform';
+
+export interface AvailableAccount {
+    name: string;
+    description?: string;
+    is_visible?: boolean;
+    is_disabled?: boolean;
+    platform?: string;
+    market_type?: 'all' | 'financial' | 'synthetic';
+    icon: string;
+    availability: string;
+    link_to?: string;
+}
+
+export const available_traders_hub_cfd_accounts: AvailableAccount[] = [
```

Comment:
i couldn't understand this logic. could you pls explain why do we need this object?

Reply:
@amina-deriv, Please read through the file `packages/appstore/src/components/cfds-listing/index.tsx` starting from line `74`, all the way down